kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -42,6 +42,12 @@ data:
           # When the setting is set to false, only the kube events with !normal event type will be collected
           enabled = false
           # When this is enabled (enabled = true), all kube events including normal events will be collected
+       #[log_collection_settings.schema]
+          # In the absense of this configmap, default value for containerlog_schema_version is "v1"
+          # Supported values for this setting are "v1","v2"
+          # See documentation for benefits of v2 schema over v1 schema before opting for "v2" schema
```

Comment:
probable, we should we add public doc  link here otherwise "see documentation" will be confusing, right?